caampr-backend/src/main/java/com/jelistan/caampr/lambda/provider/GearListProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.jelistan.caampr.lambda.provider;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBQueryExpression;
+import com.jelistan.caampr.lambda.model.Gear;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class GearListProvider implements GearProvider{
```

Comment:
`GearListProvider` may be a bit too narrow here.  I think this should probably just be `GearProviderImpl` or `DynamnoGearProvider` (dynamo implementation of the gear provider interface) as it should provide methods to interact with all facets of gear (fetch single, create, fetch list, etc)

Reply:
renamed, added getgear method that takes gearId as param